vendor/assets/stylesheets/bootstrap_toggle.css

```diff
@@ -25,4 +25,4 @@
 .toggle-off.btn-sm{padding-left:20px}
 .toggle.btn-xs{min-width:35px;min-height:22px}
 .toggle-on.btn-xs{padding-right:12px}
-.toggle-off.btn-xs{padding-left:12px}
```

Comment:
I think this was just some other tests that were in master failing - I wrote a couple bad integration tests that randomly fail, which is rude. You can probably fix by pulling down changes in master here -- try `git fetch upstream`.